### PR TITLE
Add a few comments, rename variable

### PR DESCRIPTION
**Pull request by vondele**

### CODE REVIEW

File: scoreWDL.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def __init__(self, filenames: list[str]):
         self.filenames = filenames
 
     def load_json(self) -> dict[str, int]:
+        """Load the json, which describes with the key the position, and the value the count of these positions"""
```
Comment (robertnurnberg) [inline scoreWDL.py:43]: does not read that well atm, maybe insert `with` before `the value`?

Comment (vondele) [reply scoreWDL.py]: yeah, new attempt.

File: scoreWDL.py [hunk 2]
```diff
@@ -40,6 +40,9 @@ def __init__(self, filenames: list[str]):
         self.filenames = filenames
 
     def load_json(self) -> dict[str, int]:
+        """Load the json, in which the key describes the position (result, move, material),
```
Comment (robertnurnberg) [inline scoreWDL.py:43]: could make this consistent with readme: `outcome, move, material, eval`

Comment (vondele) [reply scoreWDL.py:43]: right now it is consistent with the code a bit below, that refers to this as 'result'

Comment (vondele) [reply scoreWDL.py:43]: ah, right.

**Comment by robertnurnberg** [comment]
you could merge this and base the other PR on this one? (also exclude the `.png` change in the other one)

**Pull request merged**